backend/endpoints/platform.py

```diff
@@ -25,3 +80,40 @@ class Config:
 def platforms(request: Request) -> list[PlatformSchema]:
     """Returns platforms data"""
     return dbh.get_platforms()
+
+
+@protected_route(router.get, "/platforms/feed", [])
```

Comment:
What do you think about changing the endpoint to ``platforms/feed/webrcade``, just in case we will support any other kind of feed in the future?